src/main/java/zpi/algospace/controller/TaskController.java

```diff
@@ -18,7 +18,7 @@
 @RequiredArgsConstructor
 @Tag(name = "Task Controller")
 @RequestMapping({"/", "/api"})
-@CrossOrigin(origins = {"${allowed.origin}"})
+@CrossOrigin()
```

Comment:
te nawiasy niepotrzebne ale to przy rozwiazywaniu konflikotw i tak penwie wyjdzie